src/app/dashboard/dashboard-series.component.css

```diff
@@ -3,13 +3,13 @@ header {
   justify-content: space-between;
   align-items: flex-end;
   position: relative;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
buttons should be aligned per #608 but were looking like this with long series names
<img width="1066" alt="Screen Shot 2019-04-22 at 9 14 56 AM" src="https://user-images.githubusercontent.com/2250413/56505317-f527f800-64e0-11e9-8787-1e468fea535f.png">

so I fixed em up
<img width="686" alt="Screen Shot 2019-04-22 at 9 30 20 AM" src="https://user-images.githubusercontent.com/2250413/56505436-4e902700-64e1-11e9-95a9-0929a7dd5d9e.png">